config/application.rb

```diff
@@ -20,6 +20,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+require File.expand_path("../../extras/gvl_instrumentation", __FILE__)
```

Comment:
Shouldn’t this be one level of `../`, not two?

Reply:
No, because it's relative to `__FILE__`, not `__dir__`.

Reply:
`/srv/lobste.rs/.rbenv/versions/3.4.1/lib/ruby/3.4.0/bundled_gems.rb:82:in 'Kernel.require': cannot load such file -- /srv/lobste.rs/http/config/extras/gvl_instrumentation (LoadError)`

You moved this into `config/environments/production.rb` so I had to add an extra `../`.

Reply:
Argh, so sorry about that -_-. That's why I added it to every evens at first, so it would be tested, but I moved it to producton at the end because the path was production specific...